domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/ResponseRejection.java

```diff
@@ -19,21 +20,29 @@
 
     private final YesNoOption freeMediation;
 
+    @Size(max = 30, message = "may not be longer than {max} characters")
```

Comment:
suggestion: may be you can use @Pattern and validate with a regex.

Reply:
This should be same validation we have in front-end. We can not be more strict in back-end than front-end

Reply:
Data should be validated in back end too. We should be data gate keeper

Reply:
We should not generally believe on front end validations. If there are two clients and both do different validation, backend data will be corrupted.

Reply:
@karthikkannankainos , looks like you and @sabahirfan are saying same thing..

I will check what frontend validation we are doing..

Reply:
we are doing exactly same check in frontend

Reply:
@KiranVarma02 , what I meant is we should not believe on front end. If need basis we should do validation in backend too. Here we are getting a phone number, where I can completely put junk character.